src/components/proposal/ProposalChooseCall.tsx

```diff
@@ -48,7 +38,7 @@ const ProposalChooseCall: React.FC<ProposalChooseCallProps> = ({
   };
 
   const formatDate = (date: Date) => {
-    return dateformat(new Date(date), 'dd-mmm-yyyy');
+    return dateformat(new Date(date), 'dd-mmm-yyyy h:MM TT');
```

Comment:
Is this the same format as the `dd-MMM-yyyy hh:mm a` used earlier? It would be good to use the same format as much as possible.